ScreenOrientation.lockOrientation raises error

ScreenOrientation.lockOrientation raises error, however orientation is changing.
This happens after updated for [the issue](https://github.com/driftyco/ionic-native/issues/939)

ionic-native: "2.4.1"

```
console.error: EXCEPTION: Uncaught (in promise): TypeError: undefined is not an object (evaluating
            'pluginResult.then') http://172.26.9.242:8100/build/main.js:418:21
            http://172.26.9.242:8100/build/main.js:371:19 t@http://172.26.9.242:8100/build/polyfills.js:3:11330
            tryNativePromise@http://172.26.9.242:8100/build/main.js:370:31
            ionViewWillUnload@http://172.26.9.242:8100/build/main.js:62499:96
            _lifecycle@http://172.26.9.242:8100/build/main.js:8632:37
            _willUnload@http://172.26.9.242:8100/build/main.js:8577:24
            _willUnload@http://172.26.9.242:8100/build/main.js:33508:25
            _cleanup@http://172.26.9.242:8100/build/main.js:33442:33
            _trnsFinish@http://172.26.9.242:8100/build/main.js:33367:26 [native code]
            onInvoke@http://172.26.9.242:8100/build/main.js:40827:43
            run@http://172.26.9.242:8100/build/polyfills.js:3:6487 http://172.26.9.242:8100/build/main.js:33311:28
            _didFinish@http://172.26.9.242:8100/build/main.js:11497:29
            _didFinishAll@http://172.26.9.242:8100/build/main.js:11484:28
            onTransitionEnd@http://172.26.9.242:8100/build/main.js:10931:31
            onTransitionEnd@http://172.26.9.242:8100/build/main.js:6512:25
[18:08:43]  console.error: ORIGINAL STACKTRACE:
[18:08:43]  console.error: s@http://172.26.9.242:8100/build/polyfills.js:3:4220
            t@http://172.26.9.242:8100/build/polyfills.js:3:11356
            tryNativePromise@http://172.26.9.242:8100/build/main.js:370:31
            ionViewWillUnload@http://172.26.9.242:8100/build/main.js:62499:96
            _lifecycle@http://172.26.9.242:8100/build/main.js:8632:37
            _willUnload@http://172.26.9.242:8100/build/main.js:8577:24
            _willUnload@http://172.26.9.242:8100/build/main.js:33508:25
            _cleanup@http://172.26.9.242:8100/build/main.js:33442:33
            _trnsFinish@http://172.26.9.242:8100/build/main.js:33367:26 [native code]
            onInvoke@http://172.26.9.242:8100/build/main.js:40827:43
            run@http://172.26.9.242:8100/build/polyfills.js:3:6487 http://172.26.9.242:8100/build/main.js:33311:28
            _didFinish@http://172.26.9.242:8100/build/main.js:11497:29
            _didFinishAll@http://172.26.9.242:8100/build/main.js:11484:28
            onTransitionEnd@http://172.26.9.242:8100/build/main.js:10931:31
            onTransitionEnd@http://172.26.9.242:8100/build/main.js:6512:25
```

Same issue here, the method call works but it raises an error too :
```
main.js:100585 EXCEPTION: Uncaught (in promise): TypeError: Cannot read property 'then' of undefined
TypeError: Cannot read property 'then' of undefined
    at file:///android_asset/www/build/main.js:564:21
    at file:///android_asset/www/build/main.js:517:17
    at new t (file:///android_asset/www/build/polyfills.js:3:11329)
    at tryNativePromise (file:///android_asset/www/build/main.js:516:20)
    at getPromise (file:///android_asset/www/build/main.js:538:16)
    at wrapOtherPromise (file:///android_asset/www/build/main.js:559:12)
    at Function.<anonymous> (file:///android_asset/www/build/main.js:708:20)
    at Function.value [as lockOrientation] (file:///android_asset/www/build/main.js:772:53)
    at ScreenOrientationService.init (file:///android_asset/www/build/main.js:85542:85)
    at file:///android_asset/www/build/main.js:163401:38
ErrorHandler.handleError @ main.js:100585
next @ main.js:79754
schedulerFn @ main.js:81264
SafeSubscriber.__tryOrUnsub @ main.js:5600
SafeSubscriber.next @ main.js:5549
Subscriber._next @ main.js:5502
Subscriber.next @ main.js:5466
Subject.next @ main.js:10408
EventEmitter.emit @ main.js:81256
NgZone.triggerError @ main.js:82106
onHandleError @ main.js:82085
t.handleError @ polyfills.js:3
e.runGuarded @ polyfills.js:3
r @ polyfills.js:3
i @ polyfills.js:3
main.js:100590 ORIGINAL STACKTRACE:
ErrorHandler.handleError @ main.js:100590
next @ main.js:79754
schedulerFn @ main.js:81264
SafeSubscriber.__tryOrUnsub @ main.js:5600
SafeSubscriber.next @ main.js:5549
Subscriber._next @ main.js:5502
Subscriber.next @ main.js:5466
Subject.next @ main.js:10408
EventEmitter.emit @ main.js:81256
NgZone.triggerError @ main.js:82106
onHandleError @ main.js:82085
t.handleError @ polyfills.js:3
e.runGuarded @ polyfills.js:3
r @ polyfills.js:3
i @ polyfills.js:3
main.js:100591 Error: Uncaught (in promise): TypeError: Cannot read property 'then' of undefined
TypeError: Cannot read property 'then' of undefined
    at main.js:564
    at main.js:517
    at new t (polyfills.js:3)
    at tryNativePromise (main.js:516)
    at getPromise (main.js:538)
    at wrapOtherPromise (main.js:559)
    at Function.<anonymous> (main.js:708)
    at Function.value [as lockOrientation] (main.js:772)
    at ScreenOrientationService.init (main.js:85542)
    at main.js:163401
    at s (polyfills.js:3)
    at s (polyfills.js:3)
    at polyfills.js:3
    at t.invokeTask (polyfills.js:3)
    at Object.onInvokeTask (main.js:82055)
    at t.invokeTask (polyfills.js:3)
    at e.runTask (polyfills.js:3)
    at i (polyfills.js:3)
ErrorHandler.handleError @ main.js:100591
next @ main.js:79754
schedulerFn @ main.js:81264
SafeSubscriber.__tryOrUnsub @ main.js:5600
SafeSubscriber.next @ main.js:5549
Subscriber._next @ main.js:5502
Subscriber.next @ main.js:5466
Subject.next @ main.js:10408
EventEmitter.emit @ main.js:81256
NgZone.triggerError @ main.js:82106
onHandleError @ main.js:82085
t.handleError @ polyfills.js:3
e.runGuarded @ polyfills.js:3
r @ polyfills.js:3
i @ polyfills.js:3
polyfills.js:3 Unhandled Promise rejection: Cannot read property 'then' of undefined ; Zone: angular ; Task: Promise.then ; Value: TypeError: Cannot read property 'then' of undefined
    at main.js:564
    at main.js:517
    at new t (polyfills.js:3)
    at tryNativePromise (main.js:516)
    at getPromise (main.js:538)
    at wrapOtherPromise (main.js:559)
    at Function.<anonymous> (main.js:708)
    at Function.value [as lockOrientation] (main.js:772)
    at ScreenOrientationService.init (main.js:85542)
    at main.js:163401 TypeError: Cannot read property 'then' of undefined
    at file:///android_asset/www/build/main.js:564:21
    at file:///android_asset/www/build/main.js:517:17
    at new t (file:///android_asset/www/build/polyfills.js:3:11329)
    at tryNativePromise (file:///android_asset/www/build/main.js:516:20)
    at getPromise (file:///android_asset/www/build/main.js:538:16)
    at wrapOtherPromise (file:///android_asset/www/build/main.js:559:12)
    at Function.<anonymous> (file:///android_asset/www/build/main.js:708:20)
    at Function.value [as lockOrientation] (file:///android_asset/www/build/main.js:772:53)
    at ScreenOrientationService.init (file:///android_asset/www/build/main.js:85542:85)
    at file:///android_asset/www/build/main.js:163401:38
o @ polyfills.js:3
r @ polyfills.js:3
i @ polyfills.js:3
polyfills.js:3 Error: Uncaught (in promise): TypeError: Cannot read property 'then' of undefined
TypeError: Cannot read property 'then' of undefined
    at main.js:564
    at main.js:517
    at new t (polyfills.js:3)
    at tryNativePromise (main.js:516)
    at getPromise (main.js:538)
    at wrapOtherPromise (main.js:559)
    at Function.<anonymous> (main.js:708)
    at Function.value [as lockOrientation] (main.js:772)
    at ScreenOrientationService.init (main.js:85542)
    at main.js:163401
    at main.js:564
    at main.js:517
    at new t (polyfills.js:3)
    at tryNativePromise (main.js:516)
    at getPromise (main.js:538)
    at wrapOtherPromise (main.js:559)
    at Function.<anonymous> (main.js:708)
    at Function.value [as lockOrientation] (main.js:772)
    at ScreenOrientationService.init (main.js:85542)
    at main.js:163401
    at s (polyfills.js:3)
    at s (polyfills.js:3)
    at polyfills.js:3
    at t.invokeTask (polyfills.js:3)
    at Object.onInvokeTask (main.js:82055)
    at t.invokeTask (polyfills.js:3)
    at e.runTask (polyfills.js:3)
    at i (polyfills.js:3)
```

The method ScreenOrientation.lockOrientation is called this way and worked perfectly fine until ionic-native update 2.4.1.

```
        if (this.platform.is('cordova')) {
            ScreenOrientation.lockOrientation(this.getOrientation());
        }
```

--- a/src/plugins/screen-orientation.ts
+++ b/src/plugins/screen-orientation.ts
@@ -48,11 +48,10 @@ export class ScreenOrientation {
   /**
    * Lock the orientation to the passed value.
    * See below for accepted values
-   * @param orientation {string} The orientation which should be locked. Accepted values see table below.
-   * @returns {Promise<any>} returns a promise that resolves when the screen orientation is locked, and rejects when an error occurs.
+   * @param orientation {string} The orientation which should be locked. Accepted values see table above.
    */
-  @Cordova({ otherPromise: true })
-  static lockOrientation(orientation: string): Promise<string> { return; }
+  @Cordova({ sync: true })
+  static lockOrientation(orientation: string): void { }
 
   /**
    * Unlock and allow all orientations.

